src/Sql/dbo/Tables/Event.sql

```diff
@@ -16,6 +16,7 @@
     [ProviderId]             UNIQUEIDENTIFIER NULL,
     [ProviderUserId]         UNIQUEIDENTIFIER NULL,
     [ProviderOrganizationId] UNIQUEIDENTIFIER NULL,
+    [SystemUser]             TINYINT          NULL
```

Comment:
You're missing a comma at the end of this line, I don't know if it matters though.